### PR TITLE
federated aws cli support

### PR DESCRIPTION
**Pull request by gdestuynder**

### CODE REVIEW

File: rules/AWS-Federated-AMR.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    //  ],
+    //  "vpn_eis_automation" : [
+    //      "arn:aws:iam::656532927350:role/federated-aws-cli-test-may2019"
+    //    ]
+    // }
+    var options = { method: 'GET', url: configuration.aws_mapping_url };
+    request(options, function (error, response, body) {
+      if (error) throw new Error(error);
+      if (response.statusCode !== 200) {
+        console.log('Could not AWS amr mappings URL: '+response.statusCode);
```
Comment (gene1wood) [inline rules/AWS-Federated-AMR.js]: Not english

File: rules/AWS-Federated-AMR.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+      } else {
+        aws_groups = Object.keys(JSON.parse(body));
+        user.groups = user.groups || [];
+        context.idToken.amr = user.groups).filter(function(n) { return aws_groups.indexOf(n)  > -1 });
```
Comment (gdestuynder) [inline rules/AWS-Federated-AMR.js]:
```suggestion
        context.idToken.amr = user.groups).filter(function(n) { return aws_groups.indexOf(n)  > -1 ;});
```

Comment (gene1wood) [inline rules/AWS-Federated-AMR.js]: What is this `)` for after `groups`?

File: rules/AWS-Federated-AMR.js (hunk 1 above)
Comment (gdestuynder) [inline rules/AWS-Federated-AMR.js]:
```suggestion
        console.log('Could not fetch AWS amr mappings URL: '+response.statusCode);
```

**Pull request merged**